src/dotnet/UI.Blazor/Components/Pic/Pic.razor

```diff
@@ -20,7 +20,9 @@
 }
 
 <div class="pic @Class @picClass">
-    @if (hasPicture) {
+    @if (IsButton) {
```

Comment:
Looks strange. Why do you need it?

Reply:
![image](https://github.com/Actual-Chat/actual-chat/assets/72043982/84514f4b-3f1d-490b-905b-a2dc0241e028)
This is to turn picuploader into such button.

Reply:
What I want to say in general that you don't have to extend `PicUploader` and `Pic` for a concrete case
IMO it's better to use just a button and `FileUpload` for this case if possible

Reply:
Done